src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -284,6 +288,11 @@ export class PoolFormComponent extends CdForm implements OnInit {
     this.data.pgs = this.form.getValue('pgNum');
     this.setAvailableApps(this.data.applications.default.concat(pool.application_metadata));
     this.data.applications.selected = pool.application_metadata;
+    this.rbdMirroringService
+      .getPool(pool.pool_name)
+      .subscribe((resp: PoolEditModeResponseModel) => {
+        this.form.get('mirroringEnabled').setValue(resp.mirror_mode === 'pool');
+      });
```

Comment:
if you want we can even go further to get the mirroring status from the GET /api/pool endpoint itself.